### PR TITLE
minor code refactor and fixes

### PR DESCRIPTION
**Pull request by saileshkasaju**
most of the changes were done by prettier :)
Thanks for the workshop.

### CODE REVIEW

**Comment by rek** [comment]
Hey bro. This is awesome thanks! 

Though personally I dislike semicolons and using props inline. 

Semicolons are a relic of the old days and just add extra noise now.

Props inline make things slower to develop, so I like to list them on a new line so adding and removing is faster.

```
	<UncontrolledComponent
		handleBlur={this.handleBlur} 
	/>
```
```
	<UncontrolledComponent handleBlur={this.handleBlur} />
```

Eg: to add a prop to the top one, i only need to be on the line and press enter then type, or just delete that line to remove the prop. to edit the second one, it takes way more key strokes to get to the right place or you have to use the mouse, which is slower.


I would love to add your refactor if you wanna change those two style things above.

File: src/react/01 - controlled components/fullyuncontrolled.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+		return (
+			<React.Fragment>
+				<label>Fully Un-Controlled:</label>
+				<input
+					className='awesome-input'
+					type={type}
+					onBlur={this.handleBlur}
+					placeholder='Name...'
+					ref={(el) => {
```
Comment (rek) [inline src/react/01 - controlled components/fullyuncontrolled.js]: This is the old way of creating Ref's. Have a look into the 'new' React.createRef();

https://reactjs.org/docs/refs-and-the-dom.html

Comment (saileshkasaju) [reply src/react/01 - controlled components/fullyuncontrolled.js]: thanks. updated it.

**Review by rek — APPROVED** [review]
thanks!

**Pull request merged**